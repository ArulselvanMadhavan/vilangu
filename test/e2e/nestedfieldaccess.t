class Lang {
  int id;
}

class French extends Lang {
  int[] langs;                  //ref type
}

int main() {
  int[][] v;
  French f;
  f = new French();
  f.langs = new int[5];
  v = new int[3][];
  v[0] = new int[30];
  f.langs[4] = 5;  // f.langs - i32arr*
  v[0][0] = 900;
  out f.langs[4];
  out v[0][0];
}