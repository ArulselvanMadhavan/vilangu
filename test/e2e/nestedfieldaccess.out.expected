33
43
33
